ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -59,4 +60,47 @@ public struct ShadowInfo: Equatable {
 
     /// The vertical offset of the shadow for shadow 2.
     public let yTwo: CGFloat
+
+    /// The number that the figma blur needs to be divided by to properly display shadows
+    private let blurDivisor: CGFloat = 2.0
+}
+
+public class ShadowUtil {
+
+    public static func applyShadow(_ shadowInfo: ShadowInfo, for view: UIView) {
+        guard let superview = view.superview else {
+            return
+        }
+
+        let shadowView = UIView()
+
+        shadowView.frame = view.frame
+        shadowView.layer.cornerRadius = view.layer.cornerRadius
+
+        superview.insertSubview(shadowView, belowSubview: view)
```

Comment:
that's interesting if the shadow is applied for superview of the passined view, if the view resizes or moves around(animation), how does the frame get updated accordingly?

Reply:
I missed that, I can add functionality to update the shadow frame if the view's frame changes.